recipes/continuable/all/conanfile.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+from conans import ConanFile, tools
+
+
+class ContinuableConan(ConanFile):
+    name = "continuable"
+    description = "C++14 asynchronous allocation aware futures (supporting then, exception handling, coroutines and connections)"
+    topics = "asynchronous", "future", "coroutines", "header-only"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Naios/continuable"
+    license = "MIT"
+    settings = "os", "compiler"
+    no_copy_source = True
+    requires = (
+        "function2/4.1.0",
+    )
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        tools.check_min_cppstd(self, "14")
```

Comment:
```suggestion
    def configure(self):
        minimal_cpp_standard = "14"
        if self.settings.compiler.cppstd:
            tools.check_min_cppstd(self, minimal_cpp_standard)
        minimal_version = {
            "gcc": "5",
            "clang": "3.4",
            "apple-clang": "10",
            "Visual Studio": "14"
        }
        compiler = str(self.settings.compiler)
        if compiler not in minimal_version:
            self.output.warn(
                "%s recipe lacks information about the %s compiler standard version support" % (self.name, compiler))
            self.output.warn(
                "%s requires a compiler that supports at least C++%s" % (self.name, minimal_cpp_standard))
            return
        version = tools.Version(self.settings.compiler.version)
        if version < minimal_version[compiler]:
            raise ConanInvalidConfiguration("%s requires a compiler that supports at least C++%s" % (self.name, minimal_cpp_standard))
```

Here is a regular usage for c++14 on conan recipes.

Reply:
Oh i misinterpreted what `check_min_cppstd` does. 
Anyway is there a reason why there is no dedicated command to check for compiler support?

Reply:
That’s a fair amount of boilerplate

Reply:
That’s a fair amount of boilerplate